<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openpv</class>
 <widget class="QMainWindow" name="openpv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>openpv</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="grpPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Port</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="cmbPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>23</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ttyUSB0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbBaud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbMode">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>23</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8n1</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QwtDial" name="dialVolt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="upperBound">
     <double>14.000000000000000</double>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtRawData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtVolt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset theme="help">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>AboutQt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
